docs/application/dotnet/guides/applications/uiapplication/ui-app.md

```diff
@@ -48,37 +47,6 @@ system events occur. The following table lists the related methods.
 | `OnLocaleChanged()`            | This method is responsible for refreshing the display into the new language. |
 | `OnRegionFormatChanged()`      | This method is responsible for refreshing the display into the new time zone. |
 
-<a name="state_trans"></a>
-## Application States and Transitions
-
-
-The Tizen .NET application can be in one of several application states.
-
-The
-[Tizen.Applications](/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.html)
-namespace defines 5 states with corresponding state change methods. A
-state change method is triggered after each state change: whenever the
-application is created, starts running, or is paused, resumed, or
-terminated. The application must [react to each state change
-appropriately](#fundamentals).
-
-**Table: Application states**
-
-| State        | Description                              |
-|------------|----------------------------------------|
-| `READY`      | Application is launched.                 |
-| `CREATED`    | Application starts the main loop.        |
-| `RUNNING`    | Application is running and visible to the user. |
-| `PAUSED`     | Application is running but invisible to the user. |
-| `TERMINATED` | Application is terminated.               |
-
-The following figure illustrates the application state transitions.
-
-**Figure: Application state transitions**
-
-![Application state
-transitions](./media/app_state_transitions_cs.png)
```

Comment:
guides/applications/uiapplication/media/app_state_transitions_cs.png was used, but with this update, now this image is not in use.

If guides/applications/uiapplication/media/app_state_transitions_cs.png will not be used any more, how about removing it?